src/py/aspen/config/production.py

```diff
@@ -8,5 +9,4 @@ def DEBUG(self) -> bool:
 
     @property
     def AUTH0_CALLBACK_URL(self) -> str:
-        # TODO: this needs a realistic value.
-        return ""
+        return f"{os.getenv('API_URL', '')}/callback"
```

Comment:
We configure the API_URL env var in terraform

Reply:
i would rather this env variable be named `AUTH0_CALLBACK_URL` (i think `API_URL` sounds a little vague), but that's a nit pick

Reply:
This is the URL of the flask app, and the callback URL this method generates is for this endpoint:
https://github.com/chanzuckerberg/aspen/blob/trunk/src/py/aspen/app/views/auth.py#L9